src/utils/prismic.js

```diff
@@ -27,9 +27,12 @@ import * as Prismic from 'services/prismic';
 export const mapFromDocumentDataToString = (
   document: Array<Prismic.DocumentData>,
   prismicContent: Array<string>,
+  htmlContent?: boolean,
```

Comment:
This would make more sense

```suggestion
  isHtmlContent?: boolean,
```